script.js

```diff
@@ -67,58 +74,98 @@ function checkWinner() {
     [0, 4, 8],
     [2, 4, 6],
   ];
-  let winMessage = '';
+  let winMessage = "";
   winningCombinations.forEach((set) => {
     let indexA = set[0];
     let indexB = set[1];
     let indexC = set[2];
     if (
-        gameData.gameState[indexA] === "x" &&
-        gameData.gameState[indexB] === "x" &&
-        gameData.gameState[indexC] === "x"
+      gameData.gameState[indexA] === "x" &&
+      gameData.gameState[indexB] === "x" &&
+      gameData.gameState[indexC] === "x"
     ) {
       winMessage = "Player 1 wins!";
     }
     if (
-        gameData.gameState[indexA] === "o" &&
-        gameData.gameState[indexB] === "o" &&
-        gameData.gameState[indexC] === "o"
+      gameData.gameState[indexA] === "o" &&
+      gameData.gameState[indexB] === "o" &&
+      gameData.gameState[indexC] === "o"
     ) {
       winMessage = "Player 2 wins!";
     }
-    if (winMessage === '' && gameData.turns === 9){
-      winMessage = "It's a draw"
+    if (winMessage === "" && gameData.turns === 9) {
+      winMessage = "It's a draw";
     }
-  })
+  });
   return winMessage;
 }
+function chooseCursor() {
+  if (gameData.currentPlayer === "x") {
+    document.querySelector("#gameBoard").style.cursor = "url('imageAssets/tinyX.png'), auto";
+    document.querySelector("#playerOneBox").style.border = "10px solid #f9b233";
+  }
+  else if (gameData.currentPlayer === "o") {
+    document.querySelector("#gameBoard").style.cursor = "url('imageAssets/tinyO.png'), auto";
+    document.querySelector("#playerTwoBox").style.border = "10px solid #e6332a"
+  }
+}
 
 function playGame() {
+  chooseCursor();
   gameData.turns = 0;
-  startBtn.removeEventListener('click', playGame, false);
+  startBtn.removeEventListener("click", playGame, false);
   startBtn.style.opacity = "0.5";
+  resetBtn.style.opacity = "1";
   tiles.forEach((tile) => {
     gameData.gameState = ["", "", "", "", "", "", "", "", ""];
```

Comment:
I've just noticed that this line of code to reset the gameState is inside this foreach loop, which just means that it needlessly resets the state for each tile.

Could this be moved to be before the foreach?